tests/Go/Functional/BaseFunctionalTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Go\Functional;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\Process;
+
+abstract class BaseFunctionalTest extends TestCase
+{
+    protected static $projectDir = __DIR__.'/../../Fixtures/project';
+    protected static $aspectCacheDir = __DIR__.'/../../Fixtures/project/var/cache/aspect';
+    protected static $consolePath = __DIR__.'/../../Fixtures/project/bin/console';
+    protected static $frontControllerPath = __DIR__.'/../../Fixtures/project/web/index.php';
```

Comment:
Not recommended at all. In order to be compatible with static calls, such as `public static function setUpBeforeClass()` they must be static, that is, must be accessible in static context.

Best thing what I can do is to define them as constants, if you prefer that more.